rider/src/main/kotlin/com/jetbrains/rider/UnityProjectDiscoverer.kt

```diff
@@ -55,7 +55,8 @@ class UnityProjectDiscoverer(project: Project) : LifetimedProjectService(project
     // Returns false when opening a Unity project as a plain folder
     private fun isCorrectlyLoadedSolution(project: Project): Boolean {
         val solutionFile = project.solutionFile
-        return project.solutionDescription is RdExistingSolution && solutionFile.isFile && solutionFile.extension.equals("sln", true)
+        return project.solutionDescription is RdExistingSolution && solutionFile.isFile &&
+            (solutionFile.extension.equals("sln", true) || solutionFile.extension.equals("slnf", true))
```

Comment:
Same as above. Is there a scenario when we get `RdExistingSolution` and `solutionFile` is a file, not a folder, and the file extension is NOT `sln` or `slnf`?

Reply:
Just checking `RdExistingSolution` is enough.